### PR TITLE
Restyle homepage

### PR DESCRIPTION
**Pull request by yicyao**
Restyle home page

From this (old):
ezgif com-video-to-gif

To this (new):
fYVjsmfHE9Zh4HaNREu5sb

### CODE REVIEW

**Comment by nikhitashanker** [comment]
LGTM after change to image at top.

File: project/src/main/webapp/index.css [hunk 1]
```diff
@@ -57,6 +41,13 @@ h6 {
 }
 
 .about img {
-  height: 300px;
-  width: 300px;
+  height: 200px;
+  width: 200px;
+    display:block;
+    margin:auto;
```
Comment (nikhitashanker) [inline project/src/main/webapp/index.css:47]: nit: fix spacing

Comment (nikhitashanker) [reply project/src/main/webapp/index.css:47]: Alphabetize CSS here as well.

Comment (yicyao) [reply project/src/main/webapp/index.css]: done

**Pull request merged**